_posts/en/newsletters/2023-08-02-newsletter.md

```diff
@@ -0,0 +1,173 @@
+---
+title: 'Bitcoin Optech Newsletter #262'
+permalink: /en/newsletters/2023/08/02/
+name: 2023-08-02-newsletter
+slug: 2023-08-02-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Transcripts of periodic LN specification meetings:** Carla
+  Kirk-Cohen [posted][kc scripts] to the Lightning-Dev mailing list to
+  announce that the last several video conference meetings to discuss
+  changes to the LN specification were transcribed.  The transcripts are
```

Comment:
```suggestion
  announce transcriptions of the last several video conference meetings to discuss
  changes to the LN specification.  The transcripts are
```

Reply:
Leaving as-is.  I think the sentence is stronger with the key word at the end.